binderhub/static/index.css

```diff
@@ -129,6 +150,86 @@ h4 {
     font-family: "Roboto Mono", monospace;
 }
 
+.url, .badges{
+    background-color: #ffffff;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    border: 1px solid #ccc;
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+}
+
+
+.url{
+    margin-bottom:20px;
+}
+
+.url li {
+    list-style-type: none;
+    padding: 0 12px 0 12px;
+    height:30px;
+}
+
+.badges{
+    margin-bottom:40px;
+}
+
+.badges li{
+    list-style-type: none;
+}
+
+.bluedropdown{
+    background-color:#579ACA;
+    border-radius: 4px 4px 0px 0px;
+    height:35px;
+    width: 100%;
+}
+
+.bluedropdown label{
+    color:white;
+    padding: 6px 12px;
+    margin:0px;
+    width: 95%;
+}
+
+
+.badge-snippet-row{
+    width: 100%;
+}
+
+#basic-url-snippet, #markdown-badge-snippet, #rst-badge-snippet{
```

Comment:
this have duplicated style in `binderhub/static/index.css` and using 3 ids everywhere seem like a smell it should be a class, or just better use of CSS selectors.